### DISCUSSION TITLE
Does it work if I enter my client ID but disconnect my phone from Internet?

### DISCUSSION THREAD

**Discussion by BrajBliss** [TheCaduceus/WARP-UNLIMITED-ADVANCED#43] [Q&A]
Basically, I am using warp+ (connected) on my Windows laptop while the scripts run but I disconnect my Android from the Internet while the script is running. Sometimes when I turn it back on, it shows more referrals and sometimes it does not. I am not sure if those referrals are from previous iterations and just updated when I connect or if it works anyway. I cannot keep it running because the Android App crashes a lot while this script runs. That's why I was thinking does it work without the phone being connected to the Internet?

**Comment by TheCaduceus** [comment]
@vrindavan If you disconnect the device running this program from internet then it will stop working immediately.

**Reply by BrajBliss** [reply]
No, I am running it on Replit on my laptop. I am talking about my Android being disconnected from Internet. Android from where I get the ID to put in the script.

**Reply by TheCaduceus** [reply]
It will not cause any problem; you can disconnect your phone.

**Reply by BrajBliss** [reply]
Thanks.

**Reply by ch4hub** [reply]
I think is because you are connected with warp at the same time you run the script. Warp disconects you. Try run script without connect to Warp.

**Reply by BrajBliss** [reply]
Yeah. It does not affect when on the laptop, only on the mobile app. I do not use any VPN/DNS now. So not something that worries me anymore.
